src/lua/print.lua

```diff
@@ -0,0 +1,15 @@
+local M = {}
+
+M.raw_print = print
+
+_G.print = function(...)
```

Comment:
CI fails on it:

> LuajitError: ...ool/third_party/luajit/test/PUC-Rio-Lua-5.1-tests/db.lua:50: assertion failed!

(On the first assert.)

```lua
do
  local a = debug.getinfo(print)
  assert(a.what == "C" and a.short_src == "[C]")
  local b = debug.getinfo(test, "SfL")
  assert(b.name == nil and b.what == "Lua" and b.linedefined == 29 and
         b.lastlinedefined == b.linedefined + 16 and
         b.func == test and not string.find(b.short_src, "%["))
  assert(b.activelines[b.linedefined + 1] and
         b.activelines[b.lastlinedefined])
  assert(not b.activelines[b.linedefined] and
         not b.activelines[b.lastlinedefined + 1])
end
```

https://github.com/tarantool/luajit/blob/ae79d993ef95c7d19a4d0512f7722e7d6fec48df/test/PUC-Rio-Lua-5.1-tests/db.lua#L48-L59

Reply:
Worked around in `test/luajit-test-init.lua`.